frontend/src/reducers/login.js

```diff
@@ -0,0 +1,14 @@
+import {browserHistory} from 'react-router';
+
+export default function login(state = [], action) {
+    switch (action.type) {
+        case 'LOGIN_SUCCESS':
+            return state;
+        case 'LOGIN_FAIL':
+            browserHistory.push('/login');
```

Comment:
Looks strange browserHistory to be used that way, but it's fine as I'm not 100% what routing capabilities and classes is having the used version of the router.